cookbook/form/data_transformers.rst

```diff
@@ -38,17 +152,11 @@ for converting to and from the issue number and the ``Issue`` object::
 
     class IssueToNumberTransformer implements DataTransformerInterface
     {
-        /**
-         * @var ObjectManager
-         */
-        private $om;
+        private $em;
 
-        /**
-         * @param ObjectManager $om
-         */
-        public function __construct(ObjectManager $om)
+        public function __construct(ObjectManager $em)
```

Comment:
I think we shouldn't use these not-really-clear short variable names and just go with `$entityManager`. I think we should also change the typehint to `EntityManager` if we do this rename from om -> em.